CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## 0.19.5
```

Comment:
IMO we should bump the minor version, the code changes around OCI are significant.